### PR TITLE
Added README for Amazon SageMaker Components for Kubeflow Pipelines

### PR DESCRIPTION
**Pull request by IvyBazan**
Created a README with high-level description of components

### CODE REVIEW

**Comment by IvyBazan** [comment]
@googlebot I signed it!

File: components/aws/sagemaker/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Amazon SageMaker Components for Kubeflow Pipelines
+
+## Summary
+With Amazon SageMaker Components for Kubeflow Pipelines (KFP), you can create and monitor training, tuning, endpoint deployment, and batch transform jobs in Amazon SageMaker. By running Kubeflow Pipeline jobs on Amazon SageMaker, you move data processing and training jobs from the Kubernetes cluster to Amazon SageMaker’s machine learning-optimized managed service.
+
+Using the Amazon SageMaker components, each of the job steps in the pipeline workflow runs on Amazon SageMaker instead of the local Kubernetes cluster. The job parameters, status, logs, and outputs from Amazon SageMaker are still accessible from the Kubeflow Pipelines UI.
```
Comment (awcchungster) [inline components/aws/sagemaker/README.md:6]: First sentence is redundant with line above

File: components/aws/sagemaker/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+
+## Components
+Amazon SageMaker Components for Kubeflow Pipelines offer an alternative to launching compute-intensive jobs in Amazon SageMaker. These components integrate Amazon SageMaker with the portability and orchestration of Kubeflow Pipelines. The following Amazon SageMaker components have been created to integrate 6 key Amazon SageMaker features into your ML workflows. You can create a Kubeflow Pipeline built entirely using these components, or integrate individual components into your workflow as needed.
```
Comment (awcchungster) [inline components/aws/sagemaker/README.md:10]: First sentence should say in "to launching compute-intensive jobs in Kubernetes and integrates the orchestration benefits of KFP".

**Pull request merged**